### ISSUE TITLE
Enforce max values for expiration dates.

### ISSUE THREAD

**Issue by NSkelsey** [globaleaks/GlobaLeaks#1792]
It is possible to set a tip expiration date that breaks submissions with a value days=31400000 is passed into the following the server has problems.

``` python
return start_date + timedelta(seconds=seconds, minutes=minutes, hours=hours, days=days)
```

Currently exceptions like the following can be thrown:

``` bash
GlobaLeaks version: 2.64.6

OverflowError Result too large to be represented.

Traceback (most recent call last):

  File "/usr/local/lib/python2.7/dist-packages/twisted/internet/defer.py", line 1097, in _inlineCallbacks
    result = result.throwExceptionIntoGenerator(g)

  File "/usr/local/lib/python2.7/dist-packages/twisted/python/failure.py", line 389, in throwExceptionIntoGenerator
    return g.throw(self.type, self.value, self.tb)

  File "/usr/lib/python2.7/dist-packages/globaleaks/handlers/submission.py", line 435, in put
    self.request.language)

  File "/usr/local/lib/python2.7/dist-packages/twisted/python/threadpool.py", line 191, in _worker
    result = context.call(ctx, function, *args, **kwargs)

  File "/usr/local/lib/python2.7/dist-packages/twisted/python/context.py", line 118, in callWithContext
    return self.currentContext().callWithContext(ctx, func, *args, **kw)

  File "/usr/local/lib/python2.7/dist-packages/twisted/python/context.py", line 81, in callWithContext
    return func(*args,**kw)

  File "/usr/lib/python2.7/dist-packages/globaleaks/orm.py", line 108, in _wrap
    result = function(self.store, *args, **kwargs)

  File "/usr/lib/python2.7/dist-packages/globaleaks/handlers/submission.py", line 413, in create_submission
    return db_create_submission(store, token_id, request, t2w, language)

  File "/usr/lib/python2.7/dist-packages/globaleaks/handlers/submission.py", line 325, in db_create_submission
    submission.expiration_date = utc_future_date(days=context.tip_timetolive)

  File "/usr/lib/python2.7/dist-packages/globaleaks/utils/utility.py", line 276, in utc_future_date
    return utc_dynamic_date(datetime_now(), seconds, minutes, hours, days)

  File "/usr/lib/python2.7/dist-packages/globaleaks/utils/utility.py", line 262, in utc_dynamic_date
    return start_date + timedelta(seconds=seconds, minutes=minutes, hours=hours, days=days)

OverflowError: date value out of range
```

Also applies to whistleblower access expiration.

**Comment by evilaliv3** [comment]
as we added the setting -1 to mean unlimited, i would suggest to limit the max number to 365 (1 year). 

every adopter should have to choose if to be fine (1 year max is fine) or be fucked up (unlimited).
the right initiative will set the tip to max one year and carefully use the postpone feature.

in addition all the automatic postpone feature based on actions that we are implementing will help.

\cc @fpietrosanti

**Comment by evilaliv3** [comment]
I've applied the UI fix in order to prevent users from applying the wrong configuration.

We still lack the implementation of the constrant on the backend and in the next migration we should remember to set to -1 the tip_timetolive value where it is bigger than 365 days.

**Comment by NSkelsey** [comment]
Implemented as of v2.65.2
